content/doc/book/security/controller-isolation.adoc

```diff
@@ -132,8 +132,7 @@ it is possible for a build to run on the controller.
 So while not building on the built-in node is a good general practice to protect from bugs and less sophisticated attackers, an agent process taken over by a malicious user would still be able to obtain data or execute commands on the Jenkins controller.
 To prevent this, the Agent &rarr; Controller Access Control system prevents agent processes from being able to send malicious commands to the Jenkins controller.
 
-// TODO Also mention first LTS once it's known
```

Comment:
Please restore this TODO, it's still current.

Reply:
Done.  I actually rewrote it a bit, so the text describes what this filter does and a note says that prior to 2.236 (and LTS TBD), this could be disabled.  I also removed most of this discussion from this page and replaced it with a single sentence that references agent-controller-access.adoc.

But this is going to be removed from the UI, right?  Which means perhaps that separate file should not exist but that instead the contents of that file should be added as a subsection on this page?

Reply:
The current LTS release still has the UI, so we're in a transition phase. Once no current Jenkins release has the UI anymore, we can remove the documentation for it and just say the feature exists.